js/examples/llama-index-express/backend/package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "my-app",
+  "name": "llama-index-express",
   "version": "0.1.0",
   "main": "dist/index.js",
   "type": "module",
   "scripts": {
-    "build": "tsc --build tsconfig.json",
-    "start": "node --require ./dist/tracer.mjs dist/index.js",
-    "dev": "concurrently \"tsc --build tsconfig.json --watch\" \"nodemon -q dist/index.js\"",
+    "build": "tsup ./index.ts ./instrumentation.ts --format cjs --dts",
```

Comment:
Main change to the compilation to `cjs`. Without this instrumentation with nodejs doesn't work.